src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -74,6 +89,10 @@ describe( 'DefaultExporter', () => {
 
 		// Reset all mock implementations
 		jest.clearAllMocks();
+		( SiteServer.get as jest.Mock ).mockReturnValue( {
+			details: { path: '/path/to/site' },
+			executeWpCliCommand: jest.fn().mockReturnValue( { stderr: null } ),
```

Comment:
I'd recommend using `mockResolvedValue` here to mock `executeWpCliCommand` as the original function returns a promise.